targets/simple_switch_grpc/switch_runner.h

```diff
@@ -65,6 +65,7 @@ class SimpleSwitchGrpcRunner {
   int get_dp_grpc_server_port() {
     return dp_grpc_server_port;
   }
+  int mirroring_mapping_add(int mirror_id, int egress_port);
```

Comment:
Please add a comment explaining that is temporary and only until P4 Runtime supports configuring mirroring sessions.